sound.js

```diff
@@ -57,29 +83,40 @@ Sound.prototype.isLoaded = function() {
 Sound.prototype.play = function(onEnd) {
   if (this._loaded) {
     RNSound.play(this._key, (successfully) => onEnd && onEnd(successfully));
-  } else {
+    if (IsAndroid) {
+      RNSound.setSpeed(this._key, this._speed);
```

Comment:
Can you add a comment why you do this? The other people might not understand why this line is here only for Android.